shared/models/errors/advanced.yml

```diff
@@ -0,0 +1,62 @@
+type: string
+enum:
+  - address_length_exceeds_limit
+  - bank_account_already_verified
+  - bank_error
+  - custom_envelope_inventory_depleted
+  - deleted_bank_account
+  - failed_deliverability_strictness
+  - file_pages_below_min
+  - file_pages_exceed_max
+  - file_size_exceeds_limit
+  - foreign_return_address
+  - inconsistent_page_dimensions
+  - invalid_bank_account
+  - invalid_bank_account_verification
+  - invalid_check_international
+  - invalid_country_covid
+  - invalid_file
+  - invalid_file_dimensions
+  - invalid_file_download_time
+  - invalid_file_url
+  - invalid_image_dpi
+  - invalid_international_feature
+  - invalid_perforation_return_envelope
+  - invalid_template_html
+  - merge_variable_required
+  - merge_variable_whitespace
+  - payment_method_unverified
+  - pdf_encrypted
+  - special_characters_restricted
+  - unembedded_fonts
+description: >
+  A pre-defined string identifying the error category
```

Comment:
Could we please remove the "Error ", as in "Error 422:"?

Ideally, we would reproduce the tables that are present in the current documentation in the descriptions. CommonMark doesn't include tables, but it does have tabs. You could use tabs to create a table-like appearance.

Alternatively, you could check to see what flavor of markdown Redocly supports. CommonMark turns out to be the required minimum by OAS v3. Supersets of CommonMark are allowed. GFM is the most common superset and it does support tables. https://github.github.com/gfm/#tabs